File: 1212-sequential-digits/sequential-digits.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        l = []
        for i in range(1,10):
            n = i
            nn = i+1
            while n <= high and nn <= 9:
                n = n*10 + nn
                if n >= low and n <= high:
                    l.append(n)
                nn += 1
        l.sort()
        return l

File: 1212-sequential-digits/sequential-digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        vector<int>v;
        for(int i=1;i<10;i++){
            int n = i;
            int nn = i+1;
            while(n <= high && nn <= 9){
                n = n*10 + nn;
                if(n >= low && n <= high){
                    v.push_back(n);
                }
                nn++;
            }
        }
        sort(v.begin(),v.end());
        return v;
    }
};
